recipes-devtools/rust/compiler-rt_1.12.1.bb

```diff
@@ -0,0 +1,60 @@
+SUMMARY = "Rust compiler run-time"
+HOMEPAGE = "http://www.rust-lang.org"
+SECTION = "devel"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://src/compiler-rt/LICENSE.TXT;md5=bf24bca27049b52e9738451aa55771d4"
+
+SRC_URI = "\
+	https://static.rust-lang.org/dist/rustc-${PV}-src.tar.gz;name=rust \
+    "
+
+require rust.inc
+require rust-source-${PV}.inc
+
+DEPENDS += "rust-llvm-native (=${PV})"
+
+S = "${WORKDIR}/rustc-${PV}"
+
+DISABLE_STATIC = ""
+INHIBIT_DEFAULT_RUST_DEPS = "1"
+
+do_configure () {
+    # rust.inc generates these into ${WORKDIR}/mk-cfg/ but
+    # save ourselves time and patching by moving them back
+    # to the normal spot
+    mv "${WORKDIR}"/mk-cfg/*.mk "${S}/mk/cfg/" || bbfatal "couldn't move platform cfg"
```

Comment:
It could also be a good idea to push the configure arguments into a common location (presuming they're the same).

But that's less important if we're switching to cargo based builds here in 1.13.0,